src/transaction/mvcc_table.cpp

```diff
@@ -544,15 +544,21 @@ mvcctable::complete_mvcc (int tran_index, MVCCID mvccid, bool committed)
   next_tran_status_finish (next_status, next_index);
 
   assert (tran_index < m_transaction_lowest_visible_mvccids_size);
-  if (committed)
+  if (committed && is_active_transaction_server ())
```

Comment:
> Is it because of the remark in the issue?

No. It's a statement on the ATS side. While all log records are flushed during COMMIT, but not during ABORT, and mvccid on ATS can advance even in the case of ABORT. Someone might think that it is a problem because a new PTS can have a lower mvccid than ATS at the time. The remark  is about that.